version.go

```diff
@@ -2,5 +2,5 @@ package eventhub
 
 const (
 	// Version is the semantic version number
-	Version = "2.0.3"
+	Version = "2.0.4"
```

Comment:
This is only changed when there is a new version released. @jhendrixMSFT please release it after this PR is merged.

Reply:
Are you saying this file is updated to 2.0.4 after the PR has been merged and the module released?

Reply:
This file is only need to be updated before you release this lib.

Reply:
Ok great that's what I thought.  I believe @catalinaperalta was planning on making a release with this change after it's merged.